qgis-samgeo-plugin/core/python_manager.py

```
@@ -0,0 +1,439 @@
+            Qgis.MessageLevel.Info,
+        )
+
+        if progress_callback:
+            progress_callback(6, "Extracting Python...")
+
+        if os.path.exists(STANDALONE_DIR):
+            shutil.rmtree(STANDALONE_DIR)
+
+        os.makedirs(STANDALONE_DIR, exist_ok=True)
+
+        if temp_path.endswith(".tar.gz") or temp_path.endswith(".tgz"):
```

Ruff (0.16.1)

[warning] Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)